plugins/services/src/js/components/modals/CreateServiceModal.js

```diff
@@ -491,6 +497,32 @@ class CreateServiceModal extends Component {
     this.setState({ isPending: true });
   }
 
+  getExpandAdvancedSettings() {
```

Comment:
Why should the Modal know about the GeneralSection behaviour? We're passing down both `errors` and `serviceSpec` to `<CreateServiceModalForm />` would it make sense to move this logic all the way into GeneralServiceFormSection?

Reply:
I made it this way to run this code on clicking the review and run button.